src/database/db_build.js

```diff
@@ -0,0 +1,12 @@
+const fs = require ("fs");
+const db_connection = require ("./db_connection");
+
+const build_file =fs.readFileSync(__dirname + "/db_build.sql", "utf8");
```

Comment:
It's good practice to use the module path here instead of concatenation to avoid potential errors 😸